apps/serdes/Deserializer.cpp

```diff
@@ -0,0 +1,608 @@
+#include "Deserializer.h"
+#include <fstream>
+#include <iostream>
+
+std::string Deserializer::deserialize_string(const flatbuffers::String *str) {
+    return str->str();
+}
+
+Halide::Type Deserializer::deserialize_type(const Halide::Serdes::Type *type) {
+    // bits
```

Comment:
Here and elsewhere: these sort of comments don't add anything that isn't obvious from the code itself. Yes, we are getting `bits`, but I could have guessed that from the next line :-)